aclk/agent_cloud_link.c

```diff
@@ -23,6 +23,8 @@ static char *aclk_password = NULL;
 static char *global_base_topic = NULL;
 static int aclk_connecting = 0;
 int aclk_connected = 0;             // Exposed in the web-api
+usec_t aclk_session_us = 0;         // Used by the mqtt layer
+time_t aclk_session_sec = 0;        // User ny the mqtt layer
```

Comment:
```suggestion
time_t aclk_session_sec = 0;        // User by the mqtt layer
```

Reply:
You only fixed one of the two typos in the comment :)